.gitignore

```
@@ -231,6 +231,7 @@ numpy/core/src/umath/loops_unary_complex.dispatch.c
 # npysort module
 numpy/core/src/npysort/x86-qsort.dispatch.c
 numpy/core/src/npysort/x86-qsort.dispatch.*.cpp
+numpy/core/src/npysort/simd_qsort*.cpp
```

```suggestion
numpy/core/src/npysort/simd_qsort*.dispatch.*.cpp
```
to avoid ignoring the actual dispatch sources. also, would you please removes the two above paths, they are no longer needed

---

So sorry for mislead. Would please remove this line too and under the following line
https://github.com/numpy/numpy/blob/9172bdec1812e40b3bd80d0b6828b06e56ce1766/.gitignore#L210-L211
add:
```Bash
 *.dispatch.*.cpp
```
to globalize all c++ wrapped sources similar to C sources.

---

Done